### PR TITLE
feat: added a new option: --no-markdown for sarif output

### PR DESCRIPTION
**Pull request by saark-snyk**
#### What does this PR do?
Adding a new flag to options, `--no-markdown`, which removes the `markdown` property from `result.message` when using `sarif` output.

#### How should this be manually tested?
`snyk code test --sarif --no-markdown`

#### Any background context you want to provide?
Slack discussion: https://snyk.slack.com/archives/C01U14WSN73/p1642928960104800

#### What are the relevant tickets?
This is a result of a support ticket: https://snyk.zendesk.com/agent/tickets/18450

### CODE REVIEW

File: src/lib/plugins/sast/index.ts [hunk 1]
```diff
@@ -40,9 +41,13 @@ export const codePlugin: EcosystemPlugin = {
       }
       const numOfIssues = sarifTypedResult!.runs?.[0].results?.length || 0;
       analytics.add('sast-issues-found', numOfIssues);
-
       if (options.sarif || options.json) {
         if (numOfIssues > 0) {
+          if (options['no-markdown']) {
+            sarifTypedResult.runs?.[0].results?.forEach((result) => {
+              result.message = omit(result.message, ['markdown']);
```
Comment (pkey) [inline src/lib/plugins/sast/index.ts:48]: Apparently `omit` is not great for performance... Would it be better to just use either `delete` or object spread? I'm aware that this code will only be run if `no-markdown` is passed.

Comment (robcresswell) [inline src/lib/plugins/sast/index.ts:48]: Would be better not to use lodash here. You could call `delete` since you're modifying in place anyway.

**Review by robcresswell — APPROVED** [review]
Fine from SOS pov, but I'm not a huge fan; feels like a hack to work around a bug elsewhere?

**Review by pkey — APPROVED** [review]

File: help/cli-commands/code.md [hunk 2]
```diff
@@ -53,6 +53,10 @@ Print results in JSON format.
 
 Return results in SARIF format.
 
+## `--no-markdown`
+
+Should be used when using `--sarif`. Will remove the `markdown` field from the `result.message` object. Might help if parsing `arguments` is not working properly.
```
Comment (JackuB) [inline help/cli-commands/code.md:58]: Docs should be written in active voice.
Also, I'm not sure about the "Might help if" part, but up to you.

```suggestion
Removes the `markdown` field from the `result.message` object. Should be used when using `--sarif`. 
```

**Review by JackuB — APPROVED** [review]

**Pull request merged**